packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts

```diff
@@ -69,6 +69,8 @@ export interface IChoiceGroupOptionStyleProps {
   checked?: boolean;
   disabled?: boolean;
   imageIsLarge?: boolean;
+  imageHeight?: number;
+  imageWidth?: number;
```

Comment:
why are we introducing these? why are we not re-using the `imageSize` prop on the `IChoiceGroupOption`?

Reply:
Changed